### PR TITLE
Oceans Landing: tutorial banner with swimming fish

### PR DESCRIPTION
**Pull request by Erin007**
code.org/oceans 

In motion: 
oceans-swimmers

in context: 
oceans-landing

mobile: 
oceans-mobile-v1

Screen Shot 2019-11-27 at 1 05 39 PM

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/oceans.haml [hunk 1]
```diff
@@ -2,23 +2,11 @@
 title: AI Oceans
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/oceans.haml]: I think `AI For Oceans` or `AI for Oceans`?

**Review by breville — APPROVED** [review]
Awesome work!

File: pegasus/sites.v3/code.org/views/oceans_promo.haml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  %img.sea-creature{src: 'images/oceans/orange-fish.png'}
+  %img.crab{src: 'images/oceans/crab.png'}
+  .scanning-bot
+    %img.ai-bot{src: 'images/oceans/ai-bot.png'}
+  %img.ai-light{src: 'images/oceans/bot-beam-puddle.png'}
+  %img.ai-fish{src: 'images/oceans/fish-in-beam.png'}
+
+%img.mobile.mobile-feature{src: 'images/oceans/mobile-ai.png'}
+
+:javascript
```
Comment (islemaster) [inline pegasus/sites.v3/code.org/views/oceans_promo.haml:19]: :speak_no_evil: JavaScript in HAML oh no!

Should be fine for this purpose as long as we guarantee jQuery will always be available in this context (which I'm pretty sure we do absolutely everywhere).

**Comment by islemaster** [comment]
Is this change going to wreak havoc on our eyes tests?

**Comment by Erin007** [comment]
> Is this change going to wreak havoc on our eyes tests?

because the fish move?

**Comment by islemaster** [comment]
Yeah; I suppose we'll just add an ignore region for the header. :man_facepalming:

**Pull request merged**